metabrainz/templates/index/contact.html

```diff
@@ -87,22 +87,31 @@ <h3>{{ _('Forums') }}</h3>
     answer any questions you may have.', forums_url='https://community.metabrainz.org/') }}
   </p>
 
-  <h3>{{ _('IRC') }}</h3>
+  <h3>{{ _('Chat') }}</h3>
   <p>
-    {{ _('The MetaBrainz/MusicBrainz team, as well as MusicBrainz supporters, hang out in the
-    <strong><a href="%(irc_url_mb)s">#musicbrainz</a></strong> and
-    <strong><a href="%(irc_url_meb)s">#metabrainz</a></strong>
-    channels on IRC chat at irc.libera.chat. If you don\'t have an IRC client
-    installed, consider using this
-    <a href="%(irc_webchat_url)s">webchat client</a>
-    instead. Please feel free to browse the
-    <a href="%(chatlogs_url)s">IRC log archives</a>.',
+    {{ _('The MetaBrainz/MusicBrainz/ListenBrainz team and community hang out
+    on our Matrix, IRC and Discord channels. Visit the
+    <a href="https://musicbrainz.org/doc/Communication/ChatBrainz">ChatBrainz</a>
+    page and join the chat. You can also browse the
+    <a href="%(chatlogs_url)s">chat log archives</a>.',
     irc_url_mb='ircs://irc.libera.chat:6697/musicbrainz',
     irc_url_meb='ircs://irc.libera.chat:6697/metabrainz',
     irc_webchat_url='https://kiwiirc.com/nextclient/irc.libera.chat/#musicbrainz,#metabrainz',
```

Comment:
These URLs should probably be removed since they're no longer used. And the ChatBrainz one should probably be used in a similar way as these were rather than just pasting it as-is inside the string.

Reply:
Thanks for giving feedback! I was hoping someone else would come along and do everything 'proper' but it turns out this wasn't hard to do. I've made the suggested change (hopefully correctly)